### DISCUSSION TITLE
How do I restore schedule on another cluster?

### DISCUSSION THREAD

**Discussion by jargals2** [vmware-tanzu/velero#3401] [Community support Q&A] [answered]
I have "schedule" backup on my cluster. When I try to restore the schedule on another cluster, I can get all the backup by running "velero get backups", but I can't see any schedules. When I restore from the backup, it's not properly restored.

**Comment by dsu-igeek** [comment, marked as answer]
Backup storage locations (buckets essentially) can be shared by multiple Velero instances in different clusters.  Schedules are per-cluster resources and refer to things within the cluster.  We do not share schedules across clusters because this would force all clusters to have the same schedules.

You can extract the schedule yamls (schedules.velero.io resource in the Velero namespace) with kubectl and store them externally or even create them externally and keep them in something like git and apply them via kubectl.

**Reply by jargals2** [reply]
Thank you. I have another question. I have a velero schedule, and it already has about 10 backups but 6th backup status says it has "PartiallyFailed". In that case, can it restore properly from the last backup? Or will it fail since it's incremental?

**Reply by dsu-igeek** [reply]
The backups are independent of each other, so a partially failed backup will not cause problems for subsequent backups.
